### ISSUE TITLE
GPX track loading broken

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#14487]
### Describe your problem!

Today I wanted to prepare a bike caching trip by loading the corresponding GPX track into c:geo.
c:geo always failed with appropriate error toast.

At first I suspected some broken file, but it turns out it does also fail to load older tracks, which previously worked.
Seems the functionality is simply broken in 2023.08.05-NB

### How to reproduce?

- Download linked GPX tracks from GC9G3Y2 (worked for me in c:geo last week but no longer) or GC9WT7E
- Try to load the GPX as track/route in c:geo

### Actual result after these steps?

- File is shown as loaded but error toast appears "Failed to load the track/route"

### Expected result after these steps?

- Loading/Showing the tracks

### Reproducible

Yes

### c:geo Version

2023.08.05

### System information

```text
Samsung S20, Android 13
```

```
2023-08-05 11:42:46.874 31148-32521 cgeo         cgeo.geocaching  W  [RxCachedThreadScheduler-6] Problem parsing GPX Track file 'content://com.android.providers.downloads.documents/document/msf%3A1000014652': cgeo.geocaching.files.ParserException: Cannot parse .gpx file as GPX 1.1: could not parse XML
2023-08-05 11:42:46.877 31148-32521 cgeo         cgeo.geocaching  W  [RxCachedThreadScheduler-6] Problem parsing GPX Track file 'content://com.android.providers.downloads.documents/document/msf%3A1000014652': cgeo.geocaching.files.ParserException: Cannot parse .gpx file as GPX 1.0: could not parse XML
2023-08-05 11:42:46.880 31148-32521 cgeo         cgeo.geocaching  W  [RxCachedThreadScheduler-6] Problem parsing GPX Track file 'content://com.android.providers.downloads.documents/document/msf%3A1000014652': cgeo.geocaching.files.ParserException: Cannot parse .gpx file as GPX 1.1: could not parse XML (Line 1: Root element name does not match. Expected: 'http://www.topografix.com/GPX/1/1:gpx', Got: 'trk')
2023-08-05 11:42:46.883 31148-32521 cgeo         cgeo.geocaching  W  [RxCachedThreadScheduler-6] Problem parsing GPX Track file 'content://com.android.providers.downloads.documents/document/msf%3A1000014652': cgeo.geocaching.files.ParserException: Cannot parse .gpx file as GPX 1.0: could not parse XML (Line 1: Root element name does not match. Expected: 'http://www.topografix.com/GPX/1/0:gpx', Got: 'trk')
2023-08-05 11:42:46.885 31148-32521 cgeo         cgeo.geocaching  W  [RxCachedThreadScheduler-6] Problem parsing GPX Track file 'content://com.android.providers.downloads.documents/document/msf%3A1000014652': cgeo.geocaching.files.ParserException: Cannot parse .gpx file as GPX 1.0: could not parse XML
2023-08-05 11:42:46.892 31148-32521 cgeo         cgeo.geocaching  W  [RxCachedThreadScheduler-6] Problem parsing GeoJson file 'content://com.android.providers.downloads.documents/document/msf%3A1000014652': org.json.JSONException: Value <trk> of type java.lang.String cannot be converted to JSONObject
```

### Additional Information

Both files show fine in my Windows GPX editor

**Comment by moving-bits** [comment]
Sorry for the trouble, this was an unwanted side-effect of track/route loading consolidation, which did not show up in my tests. Will be fixed with PR #14488.

**Comment by Lineflyer** [comment]
Thanks....no worries, thats why I test the nightlies.
Anyway the bike tour today drained itself ;)
